rfp-proposals/filexplorer.org.md

```diff
@@ -0,0 +1,180 @@
+	* Mobile-responsive design via media queries, real-time metric updating with 0.5s polling for front-end (rather than a WebSocket implementation), aesthetically pleasing and easy to use layout.
+	* Hosted in a North American datacenter via OVH with European, Asia, and Oceania CDNs to ensure speed in loading static content across region.
+		* Specifically, for hosting, two instances will be used (SSD3, SSD2). One for hosting the main application front-end and back-end. The second for hosting the MongoDB server and documentation.
+* Open-sourced Github repo containing sub repos for each component of the platform (licensed accordingly with both MIT and Apache specifications):
+	* Sub-repo for front-end in Preact (Preact-core, Preact router, Axios)
+	* Sub-repo for back-end in Express (NodeJS, Axios)
+	* Sub-repo for Filecoin to MongoDB bridge (NodeJS)
+	* Sub-repo for documentation and milestone tracking
+* Well-documented codebase hosted on SSD2 instance using [MkDocs](https://www.mkdocs.org/).
+* User-facing documentation (hosted on separate subdomain running MkDocs), and contributing guidelines + good first issues to promote future open-source contributions from the community.
+
+## Development Roadmap
+
+The following milestones describe the development outline of filexplorer.org. Please note: the number of people working on each milestone has been omitted since this RFP response is being delivered individually.
+
+| Phase | Milestone Description | Funding | Timeline |
+|-------|-----------------------|---------|----------|
+|      1 | Setup, Design, Front-end framing                     |        $750 |   3 weeks (October 1st - 21st)       |
+|      2 | Back-end, MongoDB, Completed Implementation                      |       $1500  |   8 weeks (October 22nd - December 16)       |
+|      3 |   Documentation, API Guides, Addition to Shipyard                    |       $500  |   1 week (December 17 - 21, excluding winter festivities)       |
+|      4 |  Long-term maintenance and support                     |       $750  |  9 months (January - September 2020)        |
+
+
+#### Phase 1
+The first phase involves a few things. This involves multiple sub-milestones:
+
+1. Procuring domain name and hosting (setting up load balancing, static asset CDN, SSL certificate via Let's Encrypt)
+2. Setting up Github organization, appropriate licensing for each sub-repo, and integration with Github Actions to automatically deploy to staging and development environments on commit.
+3. High-fidelity design mock-ups in Adobe XD. These will include finalized top level summary page mock-ups, and somewhat finalized secondary page mock-ups. Secondary page mock-ups are not set in stone since I'd like to iterate and continue to add upon the displayed metrics and their styling as I work.
+4. Work-in-progress staging site deployed with:
+	1. Working basic data retrieval via back-end and MongoDB.
```

Comment:
@Anish-Agnihotri I want to make sure I'm understanding this right. Is #1 here about retrieving data from a running Filecoin node, storing it in your proposed backend, and then working with that data?